pkg/minikube/machine/start_test.go

```diff
@@ -96,3 +98,102 @@ func writeContentToTempFile(content string) (string, error) {
 
 	return path, nil
 }
+
+func TestDiskUsed(t *testing.T) {
+	ex := command.NewFakeCommandRunner()
+	ex.SetCommandToOutput(map[string]string{
+		"sh -c \"df -h /var | awk 'NR==2{print $5}'\"": "20%",
+	})
+	nonex := command.NewFakeCommandRunner()
+	nonex.SetCommandToOutput(map[string]string{
+		"sh -c \"df -h /nonexistent | awk 'NR==2{print $5}'\"": "df: /nonexistent: No such file or directory",
+	})
+
+	type args struct {
+		cr  command.Runner
+		dir string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    int
+		wantErr bool
+	}{
+		{
+			name:    "existent",
+			args:    args{ex, "/var"},
+			want:    20,
+			wantErr: false,
+		},
+		{
+			name:    "existent",
+			args:    args{nonex, "/nonexistent"},
+			want:    0,
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Logf("starting %v", tt.name)
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := DiskUsed(tt.args.cr, tt.args.dir)
+			t.Logf("err: %v\n", err)
+			t.Logf("got: %v\n", got)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("DiskUsed() error = %v, wantErr %v", err, tt.wantErr)
```

Comment:
If you don't want execution to continue you can do `t.Fatalf` instead and then you don't have to add the return yourself.